web/config/system.theme.yml

```diff
@@ -1,4 +1,4 @@
 _core:
   default_config_hash: 6lQ55NXM9ysybMQ6NzJj4dtiQ1dAkOYxdDompa-r_kk
-admin: claro
```

Comment:
Oops, I think you changed this by mistake. The admin theme should be Claro. 
We haven't built out our weather theme to support Drupal UI yet. 

**Claro theme**: 
![image](https://github.com/weather-gov/weather.gov/assets/144830694/ad421d91-c7c0-4a8f-b9f2-45b122ad65ac)


**Weather theme** - not quite there yet:
![image](https://github.com/weather-gov/weather.gov/assets/144830694/a3cbd6b6-952d-48c6-aa8b-0f4b7fe086d6)